### ISSUE TITLE
Smaller example dataset for recode_road_class exaple

### ISSUE THREAD

**Issue by hulsiejames** [udsleeds/openinfraresults#1]
Currently using an 87.2MB .geojson ( [bbbike_leeds_27_6_22.geojson](https://github.com/udsleeds/openinfra/releases/download/v0.2/bbbike_leeds_27_6_22.geojson) )

Going to create a much smaller subset to be hosted here!

**Comment by Robinlovelace** [comment]
Note: I think that hosting large input files in the releases of openinfra is fine: https://github.com/udsleeds/openinfra/releases

**Comment by Robinlovelace** [comment]
I suggest using this repo to host the *outputs* that we can deploy on a website. The *data outputs* can be hosted as releases. The less data that is pushed to Git the better!

**Comment by hulsiejames** [comment]
Good point - as suggested we can use this for hosting website examples and other outputs, releases work fine for the data. 

No worries - hope your desktop started behaving!

**Comment by hulsiejames** [comment]
This should be resolved now.

Replace whole of WY (or at least Leeds city and surrounding areas) from bbike with a subset of the origional data (~85Mb)

New subset is a 5.75km circular buffer around [Leeds city centre long, lat coordinates](https://www.latlong.net/place/leeds-west-yorkshire-uk-1813.html#:~:text=The%20latitude%20of%20Leeds%2C%20West,%C2%B0%2032'%2054.8412''%20W.), the example file is now only ~20MB. 

Example is hosted in `recode_road_class` [documentation](https://udsleeds.github.io/openinfra/reference/recode_road_class.html)

Whilst devtools::check() and gh actions workflows are working, I've just noticed there is an error in the example documentation: 
```
> u_data_large = "https://github.com/udsleeds/openinfra/releases/download/v0.2/bbbike_leeds_27_6_22.geojson"
> u_data_small = "some_url" #Add smaller example here! 
> data = sf::read_sf(u_data_large)
> output = recode_road_class(data)

#> Error in recode_road_class(data): could not find function "recode_road_class"
```
Don't know if this is something you have come across before in package development @Robinlovelace ? Most issues discussing this online are refering to importing gneral packages into Rstudio ([example](https://community.rstudio.com/t/could-not-find-function-error-even-though-fun-is-in-namespace-and-has-an-rd-file/91243/3)), not within package documentation. 

I've downloaded openinfra with devtools::install_github("udsleeds/openinfra") and the function works as expected, unlike the linked example.

**Comment by Robinlovelace** [comment]
Have you loaded the package @hulsiejames ?

**Comment by Robinlovelace** [comment]
Broader note: I think this issue probably is best placed in the openinfra package as that's where the example data + function it's for testing will be.

**Comment by Robinlovelace** [comment]
openinfra repo I mean..
